### PR TITLE
Add an API intermediate layer, add nimona stores

### PR DESCRIPTION
**Pull request by geoah**
I've introduced an API layer (better names welcome) that separates the handlers from the store.
This should allow the API to perform the nessesary actions on both the nimona stores as well as the meridian one.

@sirodoht this should also allow us to work on both sides of the API at the same time as we can mock the API.
Please review this PR so far to make sure you're happy with the direction and that we're aligned on how this can work.

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -1,74 +1,84 @@
...
+	"moul.io/chizap"
+
+	"nimona.io"
+
+	"github.com/sirodoht/meridian/internal"
 )
 
 func main() {
-	debugMode := os.Getenv("DEBUG")
+	debugMode, _ := strconv.ParseBool(os.Getenv("DEBUG"))
```
Comment (sirodoht) [inline cmd/server/main.go:23]: oh nice, taking notes

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
-# binary
+# binaries
 meridian
-.envrc
+
+# temporary
+tmp
+
+# local development
+.air.toml
```
Comment (sirodoht) [inline .gitignore:12]: @geoah do you want to share your `.air.toml` too?

Comment (geoah) [reply .gitignore:12]: @sirodoht if you have [air](https://github.com/cosmtrek/air), you should be able to just do `air init` to generate one.
It's not a mandatory thing, I know you're using modd, but I was having issues where killing modd left the server running on the background and had to hunt for it and kill it.

Comment (sirodoht) [reply .gitignore:12]: oh interesting, ok I'll test it.

**Pull request merged**